integration-tests/features/steps/server_api.py

```diff
@@ -132,26 +132,29 @@ def check_submit_feedback_without_valid_values(context, url):
         "feedback_type": True,
         "ecosystem": None
     }
-    context.response = requests.post(context.coreapi_url + url,
-                                     headers=authorization(context),
+    parms = {'user_key': context.three_scale_preview_user_key}
+    context.response = requests.post(context.core_v2_api_url + url,
```

Comment:
`core_v2_api_url` will target production during the production run , otherwise it will target staging by default